src/Api/AdminConsole/Public/Controllers/MembersController.cs

```diff
@@ -119,7 +122,9 @@ public async Task<IActionResult> List()
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     public async Task<IActionResult> Post([FromBody] MemberCreateRequestModel model)
     {
-        var associations = model.Collections?.Select(c => c.ToCollectionAccessSelection());
+        (await _applicationCacheService.GetOrganizationAbilitiesAsync())
+            .TryGetValue(_currentContext.OrganizationId.Value, out var organizationAbility);
```

Comment:
I added a `GetOrganizationAbilityAsync(orgId)` method you can use here instead. 😁 and anywhere else in this PR that you've called this method.

Reply:
Sneaky! I like it - thank you!